packages/core/test/uiSchema_test.js

```diff
@@ -533,7 +533,10 @@ describe("uiSchema", () => {
     expect(node.querySelector("div.help-block").textContent).eql("plop");
   });
 
-  describe("ui:focus", () => {
+  // Skipping due to JSDOM regression introduced in 15.2.0
+  // <input autofocus> no longer sets activeElement correctly
+  // https://github.com/jsdom/jsdom/issues/3041
+  describe.skip("ui:focus", () => {
```

Comment:
Can we instead just use a JSDOM version less than 15.2.0? It's fine to have workarounds to work around JSDOM regressions, but I don't really want to skip tests because of regressions like https://github.com/jsdom/jsdom/issues/3041.

Reply:
Sure, I reverted to jsdom 15.1.1. Would be nice to upgrade to latest jsdom, since I was thinking about migrating core to jest which uses jsdom 16, so we use the same test runner in all packages, but I understand your point.

Reply:
@epicfaace nevermind, i figured it out :) similar issue as to the other test